### PR TITLE
Notes thomas barrett

### PR DESCRIPTION
**Pull request by tsbarrett89**

### CODE REVIEW

**Comment by laurelsilva** [comment]
Hey @tsbarrett89 👋🏽 great job presenting key takeaways from each lesson! Also, great intuition on creating a markdown file for each lesson. That's the format that we tend to follow.

I have a couple of suggestions that can improve the notes:

For the [README.md](https://github.com/eggheadio-projects/create-amazing-animations-with-greensock-notes/blob/master/README.md) you need to add the title of course, course illustration (you can get this from the course page), About (you can get this from instructors page on egghead), a description of the course, a "what you'll learn" section, and a table of contents with links to each markdown file.

Consider adding an [Emoji Legend](https://github.com/eggheadio-projects/build-an-app-with-the-AWS-cloud-development-kit-notes#emoji-legend-) to your notes. This will inform readers of what they can spect from each section.

An example to follow is the notes on [Build an App with the AWS Cloud Development Kit](https://github.com/eggheadio-projects/build-an-app-with-the-AWS-cloud-development-kit-notes) course. Please take a look.

You don't need to at the Contributors section; we'll add that part.

File: 01-Setup-GreenSock-as-a-Module-with-Parcel.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Setup GreenSock As A Module With Parcel
+
+📹 [Video](https://egghead.io/courses/create-amazing-animations-with-greensock)
```
Comment (laurelsilva) [inline 01-Setup-GreenSock-as-a-Module-with-Parcel.md]: We can link to the course from the [README.md](https://github.com/eggheadio-projects/create-amazing-animations-with-greensock-notes/blob/master/README.md). Instead lets link to the lesson. 

For example: `https://egghead.io/lessons/greensock-setup-greensock-as-a-module-with-parcel`

File: 01-Setup-GreenSock-as-a-Module-with-Parcel.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Start by creating a folder for your new project, then add [parcel bundler](https://parceljs.org/getting_started.html) as a dev dependency
+
+    > yarn add -D parcel-bundler
```
Comment (laurelsilva) [inline 01-Setup-GreenSock-as-a-Module-with-Parcel.md]: This is great. I recommend only using the fenced code blocks -- they're easier and only they support syntax highlighting. Please refer to this [example](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting).

File: 01-Setup-GreenSock-as-a-Module-with-Parcel.md (hunk 1 above)
Comment (laurelsilva) [reply 01-Setup-GreenSock-as-a-Module-with-Parcel.md]: Let's do this for other lessons.

File: 07-Animate-From-a-Variable-Point-with-from-and-fromTo-in-Greensock.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Use TweenMax.from
+    - [TweenMax.from docs](https://greensock.com/docs/v2/TweenMax/static.from())
+    - very similar to the TweenMax.to() used before, except the properties will be where the animation starts, and its current position will be where it ends.
+    ```
```
Comment (laurelsilva) [inline 07-Animate-From-a-Variable-Point-with-from-and-fromTo-in-Greensock.md]: Let's optimize to keep our markdown formatted. Code and Syntax Highlighting should not be indented. This applies for the rest of the lessons.

File: 01-Setup-GreenSock-as-a-Module-with-Parcel.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Setup GreenSock As A Module With Parcel
```
Comment (laurelsilva) [inline 01-Setup-GreenSock-as-a-Module-with-Parcel.md]: Great job adding a title! Let's do this for the rest of the lessons.

**Review by laurelsilva — CHANGES_REQUESTED** [review]

File: 01-Setup-GreenSock-as-a-Module-with-Parcel.md (hunk 4 above)
Comment (laurelsilva) [reply 01-Setup-GreenSock-as-a-Module-with-Parcel.md]: A note on the lesson slug. Let's use kebab case. 
`01-Setup-GreenSock-as-a-Module-with-Parcel.md` --> `01-setup-greensock-as-a-module-with-parcel.md`

**Comment by lsminter** [comment]
Adding a `Go to Next Lesson` or `Go to Previous Lesson` buttons at the bottom of your notes page is great to have. So it would be like [Go to next lesson](github link to next lesson). Makes it very easy to navigate for people that are using the noes on github and not the notes on the lesson page.

**Pull request merged**